website/docs/r/stack_hci_virtual_hard_disk.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_stack_hci_virtual_hard_disk"
+description: |-
+  Manages an Azure Stack HCI Virtual Hard Disk.
+---
+
+# azurerm_stack_hci_virtual_hard_disk
+
+Manages an Azure Stack HCI Virtual Hard Disk.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_stack_hci_storage_path" "example" {
+  name                = "example-sp"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  custom_location_id  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ExtendedLocation/customLocations/cl1"
```

Comment:
Should this referencing the id from `azurerm_extended_location_custom_location`?

Reply:
The custom location is generated by API backend during HCI deployment, so this should reference to an `azurerm_extended_location_custom_location` data source, which is not supported for now, so the plain text is used here.